components/center_message_lock/index.tsx

```diff
@@ -93,7 +93,7 @@ export default function CenterMessageLock(props: Props) {
     let cta = (
         <button
             className='btn btn-primary'
-            onClick={notifyAdmin}
+            onClick={(e) => notifyAdmin(e, 'center_message_lock')}
```

Comment:
What does CenterMessageLock refer to? Is this is they are over the limits in posts?

Reply:
@marianunez yes that's correct

Reply:
Ok, so maybe we can do something like:
```suggestion
            onClick={(e) => notifyAdmin(e, 'center_channel_posts_over_limit_banner')}
```
?